scripts/prepublish.sh

```diff
@@ -1,5 +0,0 @@
-echo "> Start transpiling ES2015"
```

Comment:
We're not doing this any more? Oh wait. I see it now...this is being done with rollup?

Reply:
@ccuilla Yup, Rollup.js 👍  Gives us more control & can create multiple target builds, another big reason is we get a better bundle size. Current bundle/rollup setup can be further improved as well.